tests/mcpdft/test_sapdft_pdm.py

```diff
@@ -20,31 +20,42 @@
 H        0.000000000     -1.688293885      1.206105691
 H        0.000000000      1.250242874     -1.655874372
 '''
+mol_h2o = gto.M(atom = geom_h2o, basis = 'aug-cc-pVDZ', symmetry='c2v', output='/dev/null', verbose=0)
+mol_furan_cation = gto.M(atom = geom_furan, basis = 'sto-3g', charge=1, spin=1, symmetry=False, output='/dev/null', verbose=0)
 def get_h2o(iroots=3):
```

Comment:
FIXME: pass "mol" or replace mol -> mol_h2o below

Reply:
Sorry, put this comment on the wrong file at first. test_cmspdft_pdm.py is fine.